docs/content/dagster-plus/managing-deployments/branch-deployments/using-branch-deployments-with-github.mdx

```diff
@@ -30,87 +33,108 @@ To complete the steps in this guide, you'll need:
 
-Copy the following files to your repository (the linked files are shown in the Hybrid repo, there are equivalents in the Serverless repo).
+<TabGroup>
+<TabItem name="Hybrid">
 
-- [`dagster_cloud.yaml`](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart/blob/main/dagster_cloud.yaml)
-- [`.github/workflows/dagster-cloud-deploy.yml`](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart/blob/main/.github/workflows/dagster-cloud-deploy.yml)
+### Hybrid
 
-In the next step, you'll modify these files to work with your Dagster+ setup.
+1. Add the following files to the repository:
 
----
+   - [`dagster_cloud.yaml`](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart/blob/main/dagster_cloud.yaml)
+   - [`.github/workflows/dagster-cloud-deploy.yml`](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart/blob/main/.github/workflows/dagster-cloud-deploy.yml)
 
-## Step 4: Configure the GitHub repository
+2. In the [`dagster_cloud.yaml` file](/dagster-plus/managing-deployments/dagster-cloud-yaml), replace `build.registry` with the registry used by the [agent you created in Step 2](#step-2-create-and-configure-an-agent). For example:
 
-In this section, you'll:
+   ```yaml
+   # dagster_cloud.yaml
 
-1. [Add the agent registry to `dagster_cloud.yaml`](#step-41-add-the-agent-registry-to-dagster_cloudyaml)
-2. [Configure GitHub Action secrets](#step-42-configure-github-action-secrets)
-3. [Update GitHub Workflow files](#step-43-update-github-workflow-files)
-4. [Verify Action runs](#step-44-verify-action-runs)
+   locations:
+     - location_name: example_location
+       code_source:
+         python_file: repo.py
+       build:
+         directory: ./example_location
+         registry: 764506304434.dkr.ecr.us-east-1.amazonaws.com/branch-deployment-agent
+   ```
 
-### Step 4.1: Add the agent registry to dagster_cloud.yaml
+3. In the `dagster-cloud-deploy.yml` file, uncomment the `step` associated with your registry type. This step is required to set up Docker registry access.
 
-<Note>
-  If you're using Serverless deployment, this step isn't required.{" "}
-  <a href="#step-42-configure-github-action-secrets">Skip to the next step</a>.
-</Note>
+   For example, for an Amazon ECR registry, you'd uncomment the following portion of the file:
+
+   ```yaml
+   jobs:
+     dagster-cloud-deploy:
+       steps:
+         - name: Configure AWS credentials
+           uses: aws-actions/configure-aws-credentials@v1
+           with:
+             aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+             aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+             aws-region: ${{ secrets.AWS_REGION }}
+   ```
+
+4. Save and commit the files to the repository.
+
+</TabItem>
+<TabItem name="Serverless">
+
+### Serverless
```

Comment:
I don't believe the user needs to change anything in these files if they're using Serverless, but confirmation would be good

Reply:
The only thing I see that they could maybe want to change is uncommenting these lines to pass env vars 

https://github.com/dagster-io/dagster-cloud-serverless-quickstart/blob/main/.github/workflows/deploy.yml#L76

https://github.com/dagster-io/dagster-cloud-serverless-quickstart/blob/main/.github/workflows/branch_deployments.yml#L74

Reply:
I know there've been questions about these parts of the files before, but I can't recall the what/why. Why do users need to uncomment these lines if they're using our recommended method of env var management in the UI?

Reply:
They don't need to comment that out if they are setting env vars in the UI - that part was for the old way of setting secrets where the values were included in the Docker image, the new way is better than that from a security standpoint

Reply:
That's right! Thanks for the refresher, Dan!